.babelrc

```diff
@@ -1,20 +1,33 @@
 {
   "presets": [
-    "latest",
     "react",
     "stage-1"
   ],
   "env": {
     "development": {
       "presets": [
+        "latest",
         "react-hmre"
       ]
     },
     "production": {
+      "presets": [
```

Comment:
Yes, since the production build needs the default CommonJS module support from the ES2015 plugin that is included in the "latest" plugin disabled for tree-shaking, I needed to:

1. Distribute the "latest" preset to all three different levels and 
2. Override the module support setting for production only.

The result is that production builds use untransformed ES6 module load syntax, which Webpack can tree-shake.  The other builds use CommonJS transformation as they did before.